solutions/standard/variables.tf

```diff
@@ -112,11 +119,133 @@ variable "en_key_ring_name" {
 variable "en_key_name" {
   type        = string
   default     = "en-key"
+  description = "The name for the key that will be created for the Event Notifications. Not used if an existing key is specfied."
+}
+
+variable "cos_key_ring_name" {
+  type        = string
+  default     = "en-cos-key-ring"
+  description = "The name of the key ring which will be created for Object Storage. Not used if supplying an existing key or if `existing_cos_bucket_name` is specified."
+}
+
+variable "cos_key_name" {
+  type        = string
+  default     = "en-cos-key"
   description = "The name of the key which will be created for the Event Notifications. Not used if supplying an existing key."
 }
 
 variable "skip_en_kms_auth_policy" {
+  type        = bool
+  description = "Whether an IAM authorization policy is created that permits all Event Notifications instances in the resource group to read the encryption key from the KMS instance. Set to `true` to use an existing policy."
+  default     = false
+}
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "existing_cos_instance_crn" {
+  type        = string
+  nullable    = true
+  default     = null
+  description = "The CRN of an IBM Cloud Object Storage instance. If not supplied, a new instance is created."
+}
+
+variable "existing_cos_bucket_name" {
+  type        = string
+  nullable    = true
+  default     = null
+  description = "The name of an existing bucket inside the existing Object Storage instance. If not supplied, a new bucket is created."
+}
+
+variable "cos_destination_name" {
+  type        = string
+  description = "The name of the Object Storage destination which to create for the storage of failed delivery events."
+  default     = "COS Destination"
+}
+
+variable "cos_bucket_name" {
+  type        = string
+  description = "The name to use when creating the Object Storage bucket for the storage of failed delivery events. Bucket names are globally unique. If `add_bucket_name_suffix` is set to `true`, a random 4 character string is added to this name to help ensure that the bucket name is unique."
+  default     = "base-event-notifications-bucket"
+}
+
+variable "skip_en_cos_auth_policy" {
+  type        = bool
+  description = "Whether an IAM authorization policy is created that permits all Event Notifications instances in the resource group to interact with your Object Storage instance. Set to `true` to use an existing policy."
```

Comment:
This description is now out of date. The auth policy is scoped to the exact EN instance and bucket

Reply:
so i guess,

> ...that permits the Event Notification instance in the resource group to interact with your Object Storage instance.